module2/homework/generateSequence/generateSequence.hpp

```diff
@@ -2,6 +2,12 @@
 #include <vector>
 
 std::vector<int> generateSequence(int count, int step) {
-    // TODO: Implement me :)
-    return {};
-}
+    if (count<=0) {return {};}
+    else {
+        std::vector<int> vec {step};
+        for (int i=1; i<count; i++){
+            vec.push_back(step * (1 + i));
+        }
+     return vec;
```

Comment:
```suggestion
        return vec;
```
That return statement is within 'else' block and should be on the same indentation level as the rest of the code in that block.